tests/drivers/uart/uart_elementary/socs/esp32s3_procpu.conf

```diff
@@ -0,0 +1 @@
+CONFIG_DUAL_UART_TEST=y
```

Comment:
I see that `ESP32-S2` and all `riscv` SoCs don´t have `CONFIG_DUAL_UART_TEST=y` set. Is that correct?